NullPointerException on Android 9

When launching Etar i get the following stack trace:

```
01-27 19:04:56.993 17039 17039 D AndroidRuntime: Shutting down VM
01-27 19:04:56.993 17039 17039 E AndroidRuntime: FATAL EXCEPTION: main
01-27 19:04:56.993 17039 17039 E AndroidRuntime: Process: ws.xsoh.etar, PID: 17039
01-27 19:04:56.993 17039 17039 E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{ws.xsoh.etar/com.android.calendar.AllInOneActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.TextView.setVisibility(int)' on a null object reference
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2944)
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3079)
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1836)
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:193)
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6703)
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:911)
01-27 19:04:56.993 17039 17039 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.TextView.setVisibility(int)' on a null object reference
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	at com.android.calendar.AllInOneActivity.setMainPane(AllInOneActivity.java:1085)
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	at com.android.calendar.AllInOneActivity.initFragments(AllInOneActivity.java:762)
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	at com.android.calendar.AllInOneActivity.onCreate(AllInOneActivity.java:375)
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:7136)
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:7127)
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1271)
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2924)
01-27 19:04:56.993 17039 17039 E AndroidRuntime: 	... 11 more
01-27 19:04:56.995   840  1789 W ActivityManager:   Force finishing activity ws.xsoh.etar/com.android.calendar.AllInOneActivity
```

Widget is working fine.

I have version 1.0.25 installed from F-Droid.

--- a/res/layout-land/all_in_one.xml
+++ b/res/layout-land/all_in_one.xml
@@ -58,6 +58,22 @@
         android:layout_height="0dip"
         android:visibility="gone"/>
 
+    <TextView
+        android:id="@+id/date_bar"
+        android:layout_width="wrap_content"
+        android:layout_height="42dip"
+        android:gravity="center_vertical"
+        android:textSize="24sp"
+        android:visibility="gone"/>
+    <TextView
+        android:id="@+id/week_num"
+        android:layout_width="wrap_content"
+        android:layout_height="42dip"
+        android:gravity="center_vertical"
+        android:paddingStart="8dip"
+        android:textSize="18sp"
+        android:visibility="gone"/>
+
     <FrameLayout
         android:id="@+id/mini_month"
         android:layout_width="0dip"

--- a/res/layout/all_in_one.xml
+++ b/res/layout/all_in_one.xml
@@ -54,6 +54,22 @@
         android:layout_height="0dip"
         android:visibility="gone"/>
 
+    <TextView
+        android:id="@+id/date_bar"
+        android:layout_width="wrap_content"
+        android:layout_height="42dip"
+        android:gravity="center_vertical"
+        android:textSize="24sp"
+        android:visibility="gone"/>
+    <TextView
+        android:id="@+id/week_num"
+        android:layout_width="wrap_content"
+        android:layout_height="42dip"
+        android:gravity="center_vertical"
+        android:paddingStart="8dip"
+        android:textSize="18sp"
+        android:visibility="gone"/>
+
     <FrameLayout
         android:id="@+id/mini_month"
         android:layout_width="0dip"

